core/db/migrate/20141217215630_update_product_slug_index.rb

```diff
@@ -0,0 +1,6 @@
+class UpdateProductSlugIndex < ActiveRecord::Migration
+  def change
```

Comment:
If you remove a unique index, and effectively recreate it: There is a time frame violations can be inserted. By default this risk should not be taken.